Dockerfile

```diff
@@ -47,11 +47,10 @@ COPY . s2e/
 RUN cd s2e-build &&                                                         \
     make -f ../s2e/Makefile S2EPREFIX=/opt/s2e install install-win
 
+# Install s2e-env
+RUN apt-get -y install python-pip sudo && \
```

Comment:
actually, I can probably remove it from here. Sudo is required in the demo image, not in the build one.